src/pages/Profile/components/SectionHeader/styles.tsx

```diff
@@ -0,0 +1,13 @@
+import styled from "styled-components";
+
+const Colors = {
+  background: "#e6b873",
+} as const;
+
+export const SectionHeader = styled.h2`
+  padding: 50px;
```

Comment:
```suggestion
  padding: 50px 15px;
```
with 50px on all sides the section is larger than the header